contracts/DappCampNFT.sol

```diff
@@ -10,26 +10,62 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 contract DappCampNFT is ERC721Enumerable, Ownable {
     uint256 public MAX_MINTABLE_TOKENS = 5;
 
+
     constructor() ERC721("DappCamp NFT", "DCAMP") Ownable() {}
 
     string[] private collection = [
-        ""
+        "Mentor",
+        "Camper",
+        "Student",
+        "Coach"
     ];
 
+
+
+    function toString(uint256 value) internal pure returns (string memory) {
```

Comment:
You can use `toString` function from `@openzeppelin/contracts/utils/Strings.sol`